configs/devnet/docker-setup-config.yaml

```diff
@@ -1,17 +1,20 @@
 defaultStake: 10000
 defaultFee: 2000
-borChainId:
-heimdallChainId:
-borBranch:
-heimdallBranch:
+borChainId: 15005
+heimdallChainId: heimdall-4052
+borBranch: develop
+heimdallBranch: develop
 contractsBranch: jc/v0.3.1-backport
 sprintSize: 64
-blockNumber: '2'
-blockTime: '0'
-numOfValidators: 2
-numOfNonValidators: 0
-ethURL: http://ganache:9545
+blockNumber: '0'
+blockTime: '2'
+numOfValidators: 1
+numOfNonValidators: 1
+ethURL: http://35.92.248.232:9545
```

Comment:
Can this be any random ip for docker setup?

Reply:
That's actually just a placeholder.
Changed it to something more meaningfully here: https://github.com/maticnetwork/matic-cli/pull/79/commits/78a864b54d8d13ffc405e0bf0e278d26d13e6d87 

Thanks

Reply:
Thanks, LGTM!